Source/WebInspectorUI/UserInterface/Models/ResourceQueryResult.js

```diff
@@ -27,7 +27,7 @@ WI.ResourceQueryResult = class ResourceQueryResult extends WI.QueryResult
 {
     constructor(resource, searchString, matches, cookie)
     {
-        console.assert(resource instanceof WI.Resource, resource);
+        console.assert(resource instanceof WI.SourceCode, resource);
```

Comment:
Why broaden this assert for `WI.SourceCode` instead of `WI.Resource`?

Reply:
`WI.OpenResourceDialog` also looks at `WI.Script` and `WI.CSSStyleSheet`